README.md

```diff
@@ -8,79 +8,47 @@ Open Enclave SDK
 [![Nightly Libcxx Testing Status](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/buildStatus/icon?job=OpenEnclave-libcxx-tests&subject=nightly%20libcxx%20testing)](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/job/OpenEnclave-libcxx-tests/)
 [![Join the chat at https://gitter.im/openenclave/community](https://badges.gitter.im/openenclave/community.svg)](https://gitter.im/openenclave/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Introduction
-------------
-
-Open Enclave (OE) is an SDK for building enclave applications in C and C++. An
-enclave application partitions itself into two components:
-1. An untrusted component (called the host) and
-2. A trusted component (called the enclave).
-
-An _enclave_ is a protected memory region that provides confidentiality for data
-and code execution. It is an instance of a Trusted Execution Environment (TEE)
-which is usually secured by hardware, for example,
-[Intel Software Guard Extensions (SGX)](https://software.intel.com/en-us/sgx).
-
-This SDK aims to generalize the development of enclave applications across TEEs
-from different hardware vendors. The current implementation provides support for
-Intel SGX as well as preview support for OP-TEE OS on ARM TrustZone. As an
-open source project, this SDK also strives to provide a transparent solution
-that is agnostic to specific vendors, service providers and choice of operating
-systems.
-
-Getting Started
----------------
-
-### Intel SGX
 
-If you would like to start developing apps with the preview Open Enclave SDK
-release, start here for instructions to install and use the SDK package:
+----
 
-- [Ubuntu 18.04 with SGX hardware](docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md)
-- [Ubuntu 16.04 with SGX hardware](docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md)
-- [Ubuntu 18.04 or 16.04 in simulation mode](docs/GettingStartedDocs/install_oe_sdk-Simulation.md)
-- [Windows Server 2016 with SGX hardware](docs/GettingStartedDocs/install_oe_sdk-Windows.md)
+The Open Enclave SDK is a hardware-agnostic open source library for developing
+applications that run inside Hardware Trusted Execution Environments, also known
+as Enclaves.
 
-If you would like to run Ubuntu 16.04 or Ubuntu 18.04 in a Hyper-V VM on SGX
-capable hardware, see
-[Setting up a Linux Hyper-V VM on Windows with SGX Support](docs/GettingStartedDocs/HyperVLinuxVMSetup.md).
+The OE SDK project [was donated to](https://www.linuxfoundation.org/press-release/2019/08/new-cross-industry-effort-to-advance-computational-trust-and-security-for-next-generation-cloud-and-edge-computing/),
+and is hosted by, the [Confidential Computing Consortium](https://confidentialcomputing.io) (CCC).
+If your company is interested in helping to shape the evolution of confidential
+computing, consider joining the CCC.
```

Comment:
I'm not sure how to reword this, I just worry that saying "company" here discourages involvement on non-corporate projects

Reply:
Projects may be donated to the CCC, regardless of whether they come from a company or not.

However, _membership_ in the CCC costs money, and thus is more geared towards companies.

Reply:
Makes sense, this is fine then